### ISSUE TITLE
Create file shortcut

### ISSUE THREAD

**Issue by randomekek** [ap/vim-readdir#7]
This commit lets you create new files while browsing. Not sure if it's useful for you (as you do a cwd command when you visit new directories).

https://github.com/randomekek/vim-readdir/commit/2d4163d959a64581a58d927b761ee531f8be3ef8

**Comment by ap** [comment]
Hmm, yeah. I can see how that makes sense for you.

I don’t like the UI though… e.g. you don’t get tab completion. I can’t say off hand how it would have to be designed so that I’d like it… Something [more… Vim-ish](http://dilbert.com/strip/1999-12-03).

Actually… I think this kinda convinces me that adding the `chdir`s was the right choice. 😊 They were a feature request that I was little skeptical of at first… but considering the effect on other Vim commands, maybe doing the `chdir` and then not needing various other explicit features was exactly the right choice.

**Comment by randomekek** [comment]
Yeah I prefer to not chdir because it's a habit from netrw. Easier to patch config than fix muscle memory... obligatory xkcd.

**Comment by ap** [comment]
[#1172](https://xkcd.com/1172/ "xkcd: Workflow")?
